tpls/runtime-template.js

```diff
@@ -60,8 +61,13 @@ function install(options) {
                   sendEvent('onUpdating');
                 }
               } break;
-
+              case 'waiting': {
+                sw.skipWaiting();
```

Comment:
Even if this exists it's relatively new and is only in recent browser version. ServiceWorkerGlobalScope has to be called for that, i.e. send `postMessage` to the ServiceWorker (as it's done in `applyUpdate()` method).